docs/csharp/language-reference/compiler-messages/cs9026.md

```diff
@@ -0,0 +1,31 @@
+---
+description: "Compiler Error CS9026"
+title: "Compiler Error CS9026"
+ms.date: 9/3/2022
+f1_keywords:
+  - "CS9026"
+helpviewer_keywords:
+  - "CS9026"
+ms.assetid: deeaeabd-cf52-4752-8371-82d4ad383ea3
```

Comment:
This is a legacy metadata tag, and no longer needed:

```suggestion
```

Reply:
That saves me from asking what this should be other than a random GUID :)